test/src/e2e_vm_tests/test_programs/address_test/src/main.sw

```diff
@@ -6,6 +6,9 @@ use std::chain::assert;
 fn main() -> bool {
     let bits = 0x8900c5bec4ca97d4febf9ceb4754a60d782abbf3cd815836c1872116f203f861;
 
+    let bits3 = 0x1000000000000000000000000000000000000000000000000000000000000000;
+    let bits4 = 0x1000000000000000000000000000000000000000000000000000000000000000;
+// 0x 0000000000000000 0000000000000000 0000000000000000 0000000000000000
```

Comment:
What's the last line for? IIRC, you can use underscores in the hard-coded value, so you could do

```suggestion
    let bits3 = 0x1000000000000000_0000000000000000_0000000000000000_0000000000000000;
    let bits4 = 0x1000000000000000_0000000000000000_0000000000000000_0000000000000000;
```

Reply:
Nice, I like the underscores. Will update.